src/datagen/generated/minecolonies/data/minecolonies/tags/items/farmer_product.json

```diff
@@ -5,6 +5,8 @@
     "#forge:seeds",
     "minecolonies:composted_dirt",
     "minecraft:melon",
-    "minecraft:coarse_dirt"
+    "minecraft:coarse_dirt",
+    "minecraft:fermented_spider_eye",
```

Comment:
feel like fermented spider eye would be a cook assistance thing

Reply:
Is that really eatable though?

Reply:
I considered that (and baker), but I wasn't sure if the cook would cook things for external use, not just for food.  But also it made sense to give them the glistering melon, and I'm justifying the spider eye because the spiders get into their fields and the farmer turns them into moonshine, or something like that.

Reply:
yeah, assistant cooks can produce things on request too.